src/main/java/com/salesforce/dynamodbv2/mt/context/impl/MtAmazonDynamoDbContextProviderImpl.java

```diff
@@ -29,12 +29,7 @@ public void setContext(String tenantId) {
 
     @Override
     public String getContext() {
-        String value = getContextMap().get(CONTEXT_KEY);
-        if (value == null || value.trim().isEmpty()) {
-            throw new IllegalStateException("no context available");
-        } else {
-            return value;
-        }
+        return getContextMap().get(CONTEXT_KEY);
```

Comment:
is it ok to return null?

Reply:
Will change this based on our discussion.